packages/server/lambdas/alfresco/searchQueryBuilder/luceneQueryBuilder.ts

```diff
@@ -0,0 +1,78 @@
+import { IMimeSearchParameter, IModifiedSearchParameter, Paging, SearchParameter } from './types';
+
+const mimeTypesMapping = {
+  MSWORD: ['"application/msword"', '"application/vnd.openxmlformats-officedocument.wordprocessingml.document"'],
+  PDF: ['"application/pdf"'],
+};
```

Comment:
You could add more mimetypes here (in future probably the list will go on):
- EXCEL: application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
- PLAINTEXT:  text/plain
- IMAGE: image/tiff